### ISSUE TITLE
Navy blue sections with black text don't pass WCAG guidelines

### ISSUE THREAD

**Issue by adelaidis** [rodzinnie/adekwatnie#17]
![Image](https://user-images.githubusercontent.com/40543343/221696995-70b2db23-089a-4382-bd7d-a0c892a48a06.png)

There is not enough contrast between black text and navy blue (#1D3557) background.

https://webaim.org/resources/contrastchecker/

![Image](https://user-images.githubusercontent.com/40543343/221697207-865ff82e-585f-48e2-af7e-dca3af35ff76.png)

ACs TBC
